src/xcode/ENA/ENA/Source/HTTPClient/Resources/TicketValidationResultTokenResource.swift

```diff
@@ -66,5 +68,5 @@ enum TicketValidationResultTokenError: LocalizedError {
 	case RTR_CERT_PIN_NO_JWK_FOR_KID
 	case RTR_CERT_PIN_MISMATCH
```

Comment:
Can these two be removed?

Reply:
we still need the RTR_CERT_PIN_MISMATCH for one of the allowlist errors, I removed the other